interactive-bots/message_info/John42506176Linux/notes.md

```diff
@@ -0,0 +1,6 @@
+Error #1:The biggest error I had with this assignment was that in the demo code, it was never specified whether to keep both methods or not. 
+Solution #1: I eventually chose to keep only the send_message function and drop the send_reply, because the check_expected_responses message only seemed to work for only one of the functions , send_message and send_reply
+Error #2:I was using python2 instead of python3 for my virtualenv and it costed a lot of errors
+Solution #2:Updated to Python3
+Error #3:check_expected_responses function was not in the StubBotTestCase clas provided, 
+Solution #4: Changed class from StubBotTestCase to BotTestCase
```

Comment:
Can you use markdown formatting for this file? You could for example add a header
"Notes", and format each error-solution pair with a bullet point. Here is a [Markdown cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) for Github's markdown.